.travis.yml

```diff
@@ -8,6 +8,7 @@ install:
   - make install-travis
 
 script:
+  - OS_GOFLAGS="-p 1" travis_wait make build
```

Comment:
How long does the build take with 1 CPU? Default `travis_wait` is 10min, so if it's significantly longer than that, bump wait.